moncli/types.py

```diff
@@ -211,4 +212,23 @@ def validate_timezone(self, value):
         try:
             pytz.timezone(value)
         except (UnknownTimeZoneError):
-            raise ValidationError('Unknown time zone "{}".'.format(value))
\ No newline at end of file
+            raise ValidationError('Unknown time zone "{}".'.format(value))
+
+class WeekType(MondayType):
+    
+    native_type = Week
+    null_value = {}
+    allow_casts = (dict,)
+
+    def _cast(self, value):
+        try:
+            return Week(value['start'],value['end'])
+        except KeyError:
+            raise ConversionError('Cannot convert value "{}" to Week.'.format(value))
+    
+    def _export(self, value):
+        if not (value.start and value.end) :
+            return self.null_value
+        start =  value.start.strftime(DATE_FORMAT) 
+        end  = value.end.strftime(DATE_FORMAT)
+        return {'start': start, 'end': end}
```

Comment:
I did not give you the correct export dict format.  Please refer to the updated snippets. Thanks!